Packs/BitcoinAbuse/Integrations/BitcoinAbuse/BitcoinAbuse.py

```diff
@@ -242,19 +252,68 @@ def fetch_indicators(params: Dict, test_module: bool):
             try:
                 indicator_id = int(indicator['rawJSON']['id'])
                 if indicator_id not in indicators_ids:
-                    indicator['fields']['cryptocurrencyaddresstype'] = 'bitcoin'
+                    update_indicator_occurrences(indicator)
                     indicators_without_duplicates.append(indicator)
                     indicators_ids.add(indicator_id)
             except ValueError:
                 demisto.debug(f'The following indicator was found invalid and was skipped: {indicator}')
 
-    if not test_module:
-        # we submit the indicators in batches
-        for b in batch(indicators_without_duplicates, batch_size=2000):
-            demisto.createIndicators(b)  # type: ignore
-        demisto.setIntegrationContext({'have_fetched_first_time': True})
-    else:
-        demisto.results('ok')
+    for indicator in indicators_without_duplicates:
+        indicator_count = indicators_name_to_count_dict.get(indicator['value'])
+        indicator['fields']['count'] = indicator_count
+        indicator['fields']['cryptocurrencyaddresstype'] = 'bitcoin'
+
```

Comment:
Do you think it is better if we will add the `cryptocurrencyaddresstype` mapping to the configuration as we do for all other mappings?
In any case, I think it is better to do this additional mapping in a designated function `map_additional_fields` or something of that sort.